transmogrifier.js

```diff
@@ -418,6 +418,9 @@ const filters =
 
 const sinks =
 {
+    null: async function (params, data) {
+        return data;
```

Comment:
should just return, no data needed

Reply:
okok i'll change it to just return.

In order to check if the sink is null within the getSinkFunction, it must be passed something that is not null, so it will be passed the entry object. This likely means that the getSinkFunction must return both the params as well as the function since the runPipeline function requires sink.params

Reply:
Ah yup I missed that. In that case your way makes the most sense 👍